Support/Multitenancy/Multiple-Ironic-conductors/apply-bmhs.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+. ./config.sh
+
+namespace=${1:-"metal3"}
+
+kubectl create ns "${namespace}"
+
+for f in bmc-*.yaml; do
+  kubectl -n "${namespace}" apply -f "$f"
+done
+
+# Wait for the BMHs to be in available state
+check_bmh_status() {
+  local namespace=$1
+  # Get the list of BMH objects and their provisioning states
+  local states
+  states=$(kubectl get bmh -n "$namespace" -o jsonpath='{range .items[*]}{.metadata.name}:{.status.provisioning.state}{"\n"}{end}')
+
+  # Loop through each BMH and check its state
+  local all_available=true
+  while IFS= read -r line; do
+    local name
```

Comment:
local variables are function specific, not scope specific. Might as well declare them early at the function, with `states` for example.